src/index.js

```diff
@@ -132,8 +133,9 @@ class Img extends Component {
     }
     this.i.src = this.sourceList[this.state.currentIndex]
 
-    if (this.props.crossorigin) {
-      this.i.crossorigin = this.props.crossorigin;
+    if (this.props.crossOrigin || this.props.crossorigin) {
+      this.i.crossOrigin = this.props.crossOrigin || this.props.crossorigin
+      this.i.crossorigin = this.props.crossOrigin || this.props.crossorigin
```

Comment:
Do we need to set crossorigin twice (i.e. both lowercase and camelcase)?

Reply:
I wasn't sure, so I just set both camel & lower case. It doesn't do any harm, but we could remove the second one I think.